__tests__/port.test.js

```diff
@@ -0,0 +1,48 @@
+const Port = require('../src/port.js');
+
+describe('Port', () => {
+    describe('with ports and ships', () => {
+        let port;
+        let leedsPort;
+        let ships;
+        let harkanShip;
+        let emmaShip;
+
+        beforeEach(() => {
+            port = new Port('myPort');
+            leedsPort = new Port('Leeds');
+            ships = jest.fn();
+            harkanShip = jest.fn();
+            emmaShip = jest.fn();
+        });
+
+
+        test('creating a port', () => {
```

Comment:
The extra line after the test('xyz....', () => {) can be removed here and below.